src/components/Article.jsx

```diff
@@ -6,6 +6,37 @@ import Comments from "./Comments";
 function Article() {
     const { article_id } = useParams()
     const [article, setArticle] = useState({})
+    const [votePending, setVotePending] = useState(false)
+
+    function handleUpVote(){
+
+        setVotePending(true)
+
+        const newVotes = article.votes ++
+        setArticle({...article, newVotes})
```

Comment:
this isn't working for the reason you think it is, if you console.log newVotes you will see newVotes is the same as old votes, but the ++ is actually incrementing the votes in the article object